frontend/packages/operator-lifecycle-manager/src/components/operand/create-operand.tsx

```diff
@@ -56,6 +56,15 @@ export const CreateOperand: React.FC<CreateOperandProps> = ({
   model,
 }) => {
   const { t } = useTranslation();
+
+  const FORM_HELP_TEXT = t(
```

Comment:
Just inline these messages instead of creating constants for them. They're only used in one place.

Reply:
It's used in two places; I'd have to duplicate it if I inline.

Reply:
I only see one? You wouldn't need it in the dependency list if it's inlined inside of `useCallback`.

If we really need to keep it this way, I'd use camelCase `formHelpText`.

Reply:
I don't think the second one is in the diff. I can rename it.